docs/scripts/uploadBenchmark.py

```diff
@@ -277,6 +286,75 @@ def execute_walk_test(
         )
 
 
+def execute_walk_file_sequential(
```

Comment:
@BryanFauble I modified the `execute_walk_test_oop` function, and I am going to test uploading files sequentially. Do you mind taking a look? I already tested it with small files to ensure that this function works and files can be uploaded.

Reply:
Yeah this looks good to me!